api/src/utils/get-snapshot.ts

```diff
@@ -67,5 +67,5 @@ function sortDeep(raw: any): any {
 }
 
 export function getVersionedHash(item: Record<string, any>): string {
-	return getSimpleHash(`${JSON.stringify(item)}_${version}`);
+	return `${hash(item)}_${version}`;
```

Comment:
Could we somehow include the version in the hash?

Reply:
It was definitely an oversight! It should probably be `hash({ ...item, version })` instead 🤔

Reply:
> update: went with hash({ item, version }) for simplicity sake 👍

Exactly how I imagined it 👍 Thanks! ❤️